### PR TITLE
docs: moving acls and jwts to sdk specific sections

### PR DESCRIPTION
**Pull request by efgpinto**
References https://github.com/lightbend/kalix-docs/issues/1665

Dependent on https://github.com/lightbend/kalix-docs/pull/1670

### CODE REVIEW

**Comment by efgpinto** [comment]
The `validate-doc-links` will not pass until the docs re-structure is in prod.

**Comment by efgpinto** [comment]
Ready for review now.

File: docs/src/modules/spring/pages/using-jwts.adoc [hunk 1]
```diff
@@ -2,134 +2,7 @@
...
-The public and private keys are optional, however you must specify at least one of them. For example, if you have a service that only needs the key to validate JWTs issued by another service, you may configure a secret for it that just contains the public key so that it is unable to sign JWTs with that key.
-
-If you only have the private key, you can also ask Kalix to extract the public key from the private key, rather than having to do it manually yourself and passing it using the `--public-key` flag:
-
-----
-kalix secrets create asymmetric <my-secret-name> \
-  --private-key /path/to/private.key \
-  --extract-public-key
-----
+This section describes the practical aspects of configuring JSON Web Tokens (JWTs) with the Java/Scala SDKs, if you are not sure what JWTs are or how they work, see https://docs.kalix.io/security/jwts.html[Understanding JWTs] first. If you already have your JWTs generated and configured for a service
```
Comment (johanandren) [inline docs/src/modules/spring/pages/using-jwts.adoc:5]: Missing something in the end of the sentence?

Comment (efgpinto) [reply docs/src/modules/spring/pages/using-jwts.adoc]: Indeed.. Fixed now, thanks 🙇

**Review by johanandren — APPROVED** [review]

**Pull request merged**